### PR TITLE
[Inbound Outbound] stabilize inbound outbound

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Stabilize:
- PSTN
- One to N calling
- Tems adhoc
- Inbound calling
# Why

Opens up all of the adhoc calling scenarios in stable

# How Tested

Built locally

### CODE REVIEW

**Review by edwardlee-msft — APPROVED** [review]

File: packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-injected-buttons-Mobile-Android-Landscape-linux.png
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-injected-buttons-Mobile-Android-Landscape-linux.png:1]: We should rewrite this test to hide most buttons suych that the custom buttons are actually visible in the test

Comment (JamesBurnside) [reply packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-injected-buttons-Mobile-Android-Landscape-linux.png:1]: (not a regresion in this Pr though)

File: packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-people-button-Desktop-Chrome-linux.png
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-people-button-Desktop-Chrome-linux.png:1]: This change intentional?

image

File: packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/custom-buttons-Mobile-Android-Landscape-linux.png
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/custom-buttons-Mobile-Android-Landscape-linux.png:1]: Where did the people button go? (It looks more correct now, but don't see where this is changed in the stablizing of the features)

File: packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/no-mics-Mobile-Android-Landscape-linux.png
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/no-mics-Mobile-Android-Landscape-linux.png:1]: Intenional that people button has become devices button?

Comment (dmceachernmsft) [reply packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/no-mics-Mobile-Android-Landscape-linux.png:1]: I'm not sure I will look at the test and find out. I would have thought that the different buttons in the composite should be as we know it.

**Comment by JamesBurnside** [comment]
Flyout has disappeared from these snapshots

image

File: packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/custom-buttons-Mobile-Android-Landscape-linux.png
Comment (dmceachernmsft) [reply packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/custom-buttons-Mobile-Android-Landscape-linux.png:1]: The people button in the new versions of the composite (I think even in legacy) got moved into the more button so it is no longer on the main control bar.

File: packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-people-button-Desktop-Chrome-linux.png
Comment (dmceachernmsft) [reply packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/CallControls.test.ts-snapshots/call-control-new-experience-people-button-Desktop-Chrome-linux.png:1]: This is caused by some CSS being applied to the first child of the sidepane. I think we added this when we added the "AddPeople" button from the copy invite link. currently hunting it down..

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**